.github/CONTRIBUTING.md

```diff
@@ -18,9 +18,9 @@ you to accept the CLA when you open your pull request.
 [Fork][fork], then clone the repository:
 
 ```
-git clone git@github.com:your_github_username/connect-kotlin.git
-cd connect-kotlin
-git remote add upstream https://github.com/bufbuild/connect-kotlin.git
+git clone git@github.com:your_github_username/buf-gradle-plugin.git
+cd buf-gradle-plugin
+git remote add upstream https://github.com/bufbuild/buf-gradle-plugin.git
 git fetch upstream
```

Comment:
"We'd love your help making Connect better!"

is also wrong above

Reply:
> We'd love your help making the plugin better!